examples/_debug/styles/color-bins-style- override-property.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>GeoJSON Layer with CARTO Tiles Example</title>
+
+    <!-- Custom CSS -->
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css"
+    />
+
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #map {
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
+  </head>
+
+  <body>
+    <section id="map"></section>
+
+    <!-- Map libraries-->
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js"></script>
+    <script src="https://unpkg.com/deck.gl@8.2.0/dist.min.js"></script>
+
+    <script src="/dist/umd/index.min.js"></script>
+
+    <script>
+      function initializePoints() {
+        carto.auth.setDefaultCredentials({ username: 'public' });
+
+        const deckMap = carto.viz.createMap({
+          basemap: 'voyager',
+          view: {
+            longitude: -3.7003859,
+            latitude: 40.4153265,
+            zoom: 12
+          }
+        });
+
+        const style = carto.viz.style.colorBins('availability_365', {
+          bins: 5,
+          method: 'quantiles',
+          // breaks: [],
+          // palette:
+          strokeWidth: 4,
+          color: '#FFF',
+          property: 'strokeColor'
+        });
+        const layer = new carto.viz.Layer('listings_madrid', style);
+        layer.addTo(deckMap);
+      }
+
+      function initializeLines() {
+        carto.auth.setDefaultCredentials({ username: 'cartoframes' });
```

Comment:
Just a comment. 👍 This is OK for debug. For public examples, we should always use 'public' user

Reply:
By the way, the example looks really nice. 

It is not a priority right now, but we would probably want to document this new feature a bit better in the Styles guide, with a note like:

>>> With color helpers, the surface of the feature is coloured by default (thus, there is an implicit visual property, which we considered the most useful one). But if you want to use the stroke, you can specify it, with something like for example:

```js
carto.viz.style.colorBins('availability_365', { strokeWidth: 4, property: 'strokeColor' });
```

![image](https://user-images.githubusercontent.com/458196/88555042-8d3b5800-d027-11ea-897e-0fcb7ef30068.png)

cc/ @borja-munoz

Reply:
Every step we take on this direction, we're near this 'advanced VL' example: https://carto.com/developers/carto-vl/examples/maps/advanced/taxi-pickups.html which is to me one of the most complex examples I remember: several properties at the same time (+ animation)